src/wallet/types.ts

```diff
@@ -259,6 +260,16 @@ export interface IStopWalletParams {
   cleanStorage?: boolean;
 };
 
+export interface DelegateAuthorityOptions {
+  anotherAuthorityAddress: string | null;
+  createAnother: boolean | null;
```

Comment:
From the previous code, this can't be `null`, right?

Reply:
You are right, not sure why I set it to `null`, fixed in 5c6285111de7755d091e7be1884dccef6aca3975